### PR TITLE
ログイン画面作ったよ

### PR DESCRIPTION
**Pull request by pikohan-suzuki**
あああああああああああああああああああああ

### CODE REVIEW

File: login/login.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<html>
+<head>
+<title>aaa</title>
+</head>
```
Comment (MyPoZi) [inline login/login.html]: <script>adawda</script>

Comment (pikohan-suzuki) [reply login/login.html]: <br><br><br>

Comment (pikohan-suzuki) [reply login/login.html]: "<br><br><br>"

Comment (pikohan-suzuki) [reply login/login.html]: <h1 style="font-weight="bold";">あああああ</h1>

Comment (MyPoZi) [reply login/login.html]: <br>

Comment (pikohan-suzuki) [reply login/login.html]: <h1 style="background= "#ffffff"">aaaaa</h1>

**Pull request merged**